pkgs/by-name/ge/genemichaels/package.nix

```
@@ -15,6 +15,14 @@ rustPlatform.buildRustPackage rec {
     hash = "sha256-qPyIC9AbRQI+inBft7Dd5R5v+tXtJcZdiV4lBIBwpyM=";
   };
 
+  patches = [
+    # TODO(@djacu): remove this if this patch is merged and the version is
+    # bumped to a release that has this change
+    # See: https://github.com/andrewbaxter/genemichaels/issues/95
+    # See: https://github.com/andrewbaxter/genemichaels/pull/96
+    ./genemichaels-only-write-if-changed.patch
```

It'd be better to fetch the patch from GitHub rather than vendoring it in-tree

```suggestion
    (fetchpatch {
      url = "https://github.com/andrewbaxter/genemichaels/commit/3c58f090888053b5165dbb0292b5974293851e2a.patch";
      hash = "sha256-vxtO9N6P6rcfGhIfagr2fT83o0lM6J1tOckxarMJkL8=";
    })
```